### PR TITLE
[DEX] Vuex Bug Fixes, Etc.

### PR DESCRIPTION
**Pull request by crishellco**
## Ticket
* n/a

## Changes
* Fix how we access state
* Add websocket network switching (for mainnet)

## Screenshots
n/a

## Code Coverage
n/a

### CODE REVIEW

File: src/components/AuthenticatedWrapper.vue [hunk 1]
```diff
@@ -91,6 +105,25 @@ export default {
   },
 
   methods: {
+    connectWebsocket() {
+      const index = Vue._installedPlugins.indexOf(VueNativeSock);
+      // Remove the Websocket plugin if it is already installed, so we can re-init and change the uri.
+      if (index > -1) {
+        this.$disconnect();
```
Comment (jsolman) [inline src/components/AuthenticatedWrapper.vue:112]: is this asynchronous? On desktop I've seen the websocket sometimes seemingly get in a bad state after switching networks occasionally, but hard to reproduce or tell what is going on.

File: src/components/Dex.vue [hunk 2]
```diff
@@ -86,45 +86,40 @@ export default {
...
     });
 
-    store.commit('setSocketOrderMatchFailed', (message) => {
+    this.$store.commit('setSocketOrderMatchFailed', (message) => {
       services.alerts.error(`Failed to Match ${(message.side === 'bid' ? 'Buy' : 'Sell')} x${message.data.quantity}. ${message.data.errorMessage}`);
       // services.neo.resetSystemAssetBalanceCache();
     });
 
-    services.neo.promptGASFractureIfNecessary();
+    this.$services.neo.promptGASFractureIfNecessary();
```
Comment (jsolman) [inline src/components/Dex.vue:99]: GAS fracture form shows up now on mobile?

**Review by jsolman — APPROVED** [review]
Couple questions, but it looks fine.

File: src/components/Dex.vue (hunk 2 above)
Comment (crishellco) [reply src/components/Dex.vue:99]: This is just copied from desktop.

File: src/components/AuthenticatedWrapper.vue (hunk 1 above)
Comment (crishellco) [reply src/components/AuthenticatedWrapper.vue:112]: No it is not.

Comment (jsolman) [reply src/components/AuthenticatedWrapper.vue:112]: The newer version of the vue websocket library allows passing the url to connect, so this isn’t necessary anymore. I’ll merge it over later tomorrow when I do merge from desktop.

File: src/components/Dex.vue (hunk 2 above)
Comment (jsolman) [reply src/components/Dex.vue:99]: Experience may be poor with fees set for some users without the gas fracture.

**Pull request merged**